src/main/java/core/basesyntax/classes/figures/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.classes.figures;
+

```

Comment:
Remove this unnecessary empty line to maintain a clean and organized code structure.